### ISSUE TITLE
pyrdp-convert crash

### ISSUE THREAD

**Issue by PLTorrent** [GoSecure/pyrdp#463]
pyrdp-convert crashes when I try to convert a pcap rdp session. I am using the latest docker image.

```
pyrdp@33df43f0ab6a:~$ pyrdp-convert --list-only rdp.pcap
[*] Analyzing PCAP 'rdp.pcap' ...
    - 10.0.0.2:55510 -> 10.1.1.1:3389 : plaintext
pyrdp@33df43f0ab6a:~$ pyrdp-convert --src 10.1.1.1 -o ~/ rdp.pcap
[*] Analyzing PCAP 'rdp.pcap' ...
Traceback (most recent call last):
  File "/opt/venv/bin/pyrdp-convert", line 8, in <module>
    sys.exit(main())
  File "/opt/venv/lib/python3.10/site-packages/pyrdp/bin/convert.py", line 99, in main
    exitCode = converter.process()
  File "/opt/venv/lib/python3.10/site-packages/pyrdp/convert/PCAPConverter.py", line 41, in process
    streams = self.listSessions()
  File "/opt/venv/lib/python3.10/site-packages/pyrdp/convert/PCAPConverter.py", line 78, in listSessions
    if self.checkSrcExcluded(client) or self.checkDstExcluded(server):
  File "/opt/venv/lib/python3.10/site-packages/pyrdp/convert/PCAPConverter.py", line 35, in checkSrcExcluded
    return len(self.srcFilter) > 0 and src not in self.srcFilter
  File "/opt/venv/lib/python3.10/site-packages/pyrdp/convert/utils.py", line 62, in __eq__
    if self.ip == other.ip and self.port == other.port:
AttributeError: 'str' object has no attribute 'ip'
```

**Comment by obilodeau** [comment]
Couple of questions:
* Does it work with the previous released version? `docker run gosecure/pyrdp:v1.2.0 [...]`
* Is it a decrypted layer 7 PDU capture, not an encrypted RDP pcap? See README for details.
* Did you check this #457?
* Can you provide the pcap or a small subset of it that can reproduce the issue?

Thanks

**Comment by obilodeau** [comment]
Feel free to re-open the issue when you can answer the questions.
